utils/copy-env-vars.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+folders=("dtfs-central-api" "external-api" "gef-ui" "portal" "portal-api" "secrets" "trade-finance-manager-api" "trade-finance-manager-ui" "utils/mock-data-loader")
+
+for folder in "${folders[@]}"; do
+    echo "Copying .env file to $folder folder..."
+    cp .env "$folder/"
```

Comment:
Could we possibly have a section here for updating `MONGODB_URI` to use `localhost` instead of `dtfs-submissions-data` when we copy the `.env` file?

Reply:
On the same though, can we also do the same for `SQL_DB_HOST`:
* `dtfs-sql` for root `.env`.
* `localhost` for all other `.envs`.

Reply:
updated